addons/website/views/website_templates.xml

```diff
@@ -772,6 +789,15 @@
                     <opt id="option_template_header_logo_hamburger" string="Logo - Hamburger" data-xmlid="website.template_header_logo_hamburger" data-widget="auto" data-variable="header-template" data-value="space" data-reload="/"/>
                 </selection>
             </list>
+            <list data-depends="!option_hamburger_menu, !option_side_navigation" string="Header Visibility">
+                <selection>
+                    <opt id="option_header_visibility_standard" string="Standard" data-xmlid="website.affix_top_menu" data-reload="/"/>
+                    <opt id="option_header_visibility_scroll" string="Scroll" data-xmlid="" data-reload="/"/>
+                    <opt id="option_header_visibility_freeze" data-depends="!option_pre_header" string="Freeze" data-xmlid="website.header_visibility_freeze" data-reload="/"/>
```

Comment:
Same here, you cannot reference `option_pre_header`, it is not a standard option